app/src/main/java/ch/epfl/sweng/rps/ui/statistics/StatisticsFragment.kt

```diff
@@ -21,15 +25,17 @@ class StatisticsFragment : Fragment() {
         container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View {
+        // for filter
         val values = arrayOf(
             "Mode Filter",
-            "3 Matches",
-            "5 Matches",
-            "12 Matches",
+            "Best of 3",
+            "Best of 5",
+            "Best of 12",
```

Comment:
In the future, we will get the game modes from an external source, so from now you should have a method called something like `getGameModes` that returns the game modes, instead of them being hardcoded.

Reply:
I have wrapped as a new method :) Thanks for your advice!